docs/src/tutorials/calibration.qmd

```diff
@@ -0,0 +1,94 @@
+
+
+``` @meta
+CurrentModule = LaplaceRedux
+```
+
+# Uncertainty Calibration
+
+# The issue of calibrated uncertainty distributions
+
+
+Bayesian methods offer a general framework for quantifying uncertainty. However, due to model misspecification and the use of approximate inference techniques,uncertainty estimates are often inaccurate: for example, a 90% credible interval may not contain the true outcome 90% of the time, in such cases the model is said to be miscalibrated. 
+
+ This problem arises due to the limitations of the model itself: a predictor may not be sufficiently expressive to assign the right probability to every credible interval, just as it may not be able to always assign the right label to a datapoint.\ Miscalibrated credible intervals reduce the trustworthyness of the  forecaster because they lead to a false sense of precision  and either overconfidence or underconfidence in the results.
```

Comment:
Note that the "\" character is ignored. If you want a line break, you can just literally add a line break in the `qmd` file

Reply:
"trustworthiness"